registry/donor/views.py

```diff
@@ -172,3 +181,29 @@ def save_note():
     db.session.commit()
     flash("Poznámka uložena.", "success")
     return redirect(url_for("donor.detail", rc=note_form.rodne_cislo.data))
+
+
+@blueprint.post("/override")
+@login_required
+def save_override():
+    override_form = DonorsOverrideForm()
+    delete = True if "delete_btn" in request.form else False
```

Comment:
Normálně tohle řešíme dvěma samostanými formuláři a oddelěnými views, ale klidně to může být i takto a do budoucna to můžeme sjednotit na jeden nebo druhý způsob.

Reply:
Přesunu mazání do jiného formuláře a pohledu, aby jedna URL nedělala dvě věci.

Reply:
Nakonec obě tlačítka nechám v jednom formuláři, protože není žádný jednoduchý způsob, jak je rozdělit (v `<tbody>` ani `<tr>` není povolený `<form>`). Případně by šel formulář udělat úplně bez tabulky a na rozložení použít flexbox, ale s tím zkusím experimentovat později.

Reply:
Vážně? Je k tomu někde nějaké info? Koukám, že tady běžně v bunkách tabulky máme formy.

Reply:
Podle standardů to určitě správně není a ani podle developer tools v prohlížeči by to nemělo fungovat, ale ve skutečnosti to nějak záhadně funguje.

![](https://i.imgur.com/2qaPdhU.png)

Firefox `<form>` hned automaticky zavře, takže by se žádná pole neměla při potvrzení odeslat, protože nejsou součástí formuláře, ale v praxi se to chová správně.
Ještě by chtělo vyzkoušet, jak se s tím poperou ostatní prohlížeče.

Reply:
Myslel jsem to tak, že form je až v `td` - tedy `<tbody><tr><td><form>form content</form></td></tr></tbody>`.

Reply:
Pokud chceme zachovat tabulku, tak alespoň hlavní form pro zadávání údajů musí být přímo v `tr`:

```html
<table>
    <tr>
        <form>    <-----
            <td><input></td>
            ...
        </form>
        <td>
            <form>
                (tlačítko pro mazání)
            </form>
        </td>
    </tr>
</table>
```

Pokud by HTML bylo nějak takto:
```html
<form>
    <table>
        <tr>
            <td><input></td>
            ...
            <td><input></form></td>
            <td>
                <form>
                    (tlačítko pro mazání)
                </form>
             </td>
        </tr>
    </table>,
```
tak by se při uzavření prvního formu automaticky uzavřela i tabulka.

Reply:
Máš pravdu, myslel jsem na form v poslední bunce tabulky pro smazani zaznamu. Moje chyba, omlouvam se :facepalm: 

Klidně to tedy nech jako jeden form přes celou tabulku.

Reply:
Později to zkusím vyřešit nějak jinak, aby se zachovalo tabulce podobné rozložení, které je úsporné na výšku, ale aby se formulář na užší obrazovce zalamoval a aby případně šlo formulář rozdělit na ukládací a mazací.